subworkflows/local/binning.nf

```diff
@@ -136,63 +120,20 @@ workflow BINNING {
 
     GUNZIP_BINS ( ch_final_bins_for_gunzip )
     ch_binning_results_gunzipped = GUNZIP_BINS.out.gunzip
-    ch_versions = ch_versions.mix(GUNZIP_BINS.out.versions.first())
+        .groupTuple(by: 0)
```

Comment:
Do you think using `size` or `groupKey` will make this grouping operation more efficient?

Reply:
Possibly, but it's not trivial as the number of bins files vary per sample. I could upstream calculate the number of bins, insert that into a meta field for the sample, and then insert that into `size`, but this would require experimentation. I've made an issue (tagged you) to investigate for a future release, as I don't think that's a blocker at the moment.